### ISSUE TITLE
Add the drive letter

### ISSUE THREAD

**Issue by bozhinov** [tenox7/listdisk#2]
Seems like a vital piece of info

**Comment by tenox7** [comment]
yes it is, but unlike on Linux it's extremely hard to get, especially for dynamic disks

perhaps a version without dynamic disk support would suffice at first

**Comment by bozhinov** [comment]
most definitely
I did a quick googling and found out it is seen as a single volume for whatever reason.

**Comment by tenox7** [comment]
FYI this is coming up. Already added in lsblk, going to add it here next.
